_episodes/06-gh-pages.md

```diff
@@ -1,92 +1,49 @@
 ---
 layout: episode
-title: "Deploying a project website or homepage to GitHub Pages"
-teaching: 0
-exercises: 20
+title: Hosting websites/homepages on GitHub Pages
+teaching: 20
+exercises: 0
```

Comment:
It seems like it's mostly exercise - is 20/0 what it should be?

Reply:
Oops, yes, I meant the exact opposite. I will adjust.